contracts/core/AMM/SportsAMMV2Manager.sol

```diff
@@ -96,6 +99,32 @@ contract SportsAMMV2Manager is Initializable, ProxyOwned, ProxyPausable {
         return knownTickets.elements.length;
     }
 
+    /// @notice gets batch of active tickets per market
+    /// @param _index start index
+    /// @param _pageSize batch size
+    /// @param _gameId to get tickets for
+    /// @param _typeId to get tickets for
+    /// @param _playerId to get tickets for
+    /// @return activeTickets
+    function getActiveTicketsPerMarket(
```

Comment:
I wouldnt say this is `activeTickets`, its just `ticketsPerMarket`.
As we dont maintain active vs resolved tickets per market.